source/backend/discovery/src/lib/additionalRelationships/createLookUpMaps.js

```diff
@@ -50,45 +56,29 @@ function createLookUpMaps(resources) {
                     resourceId);
                 break;
             case AWS_ELASTICSEARCH_DOMAIN:
-                if(configuration.endpoint != null) dbUrlToIdMap.set(configuration.endpoint, id)
-                Object.values(configuration.endpoints ?? []).forEach(endpoint => dbUrlToIdMap.set(endpoint, id));
-                break;
             case AWS_OPENSEARCH_DOMAIN:
-                if(configuration.Endpoint != null) dbUrlToIdMap.set(configuration.Endpoint, id)
-                Object.values(configuration.Endpoints ?? []).forEach(endpoint => dbUrlToIdMap.set(endpoint, id));
+                const endpoints = configuration.endpoints ?? configuration.Endpoints ?? [];
```

Comment:
this is quite strange - is it due to old naming conventions?

Reply:
If it is required I feel like there should be some specific tests which hit these various branches? same below with dnsname and dNSName. Those tests might already exist I just couldn't see them

Reply:
Yeah, Elasticsearch uses camelCase and OpenSearch uses UpperCamelCase. The variable naming conventions returned by various services differ quite considerably. Some resource types even mix both casing strategies in the same resource.

Reply:
> If it is required I feel like there should be some specific tests which hit these various branches? same below with dnsname and dNSName. Those tests might already exist I just couldn't see them

Yeah, these tests exist, they're in the `test/additionalRelationships.js` file, specifically the ones around lambda and ECS variables.